data/hai/hai outfits.txt

```diff
@@ -157,7 +158,8 @@ outfit "Ionic Blaster Prototype"
 		"shield damage" 70
 		"hull damage" 30
 		"ion damage" 1
-	description "The Unfettered conflict with the rest of the Hai has put pressure on them to gain a technological edge, despite their limited resources. The Ionic Blaster is a result of that pressure. It is much smaller than the venerable Ion Cannon, but is still able to apply the same weapon jamming and energy draining effects, while also doing more raw damage for its size."
+		"scrambling damage" 1
+	description "The Unfettered conflict with the rest of the Hai has put pressure on them to gain a technological edge, despite their limited resources. The Ionic Blaster is a result of that pressure. It is much smaller than the venerable Ion Cannon, but is still able to apply the same scrambling and energy draining effects, while also doing more raw damage for its size."
```

Comment:
This description change seems unnecessary to me. It's not wrong the way it is, and if anything becomes less descriptive by instead of saying that it jams weapons, saying that it scrambles.
```suggestion
	description "The Unfettered conflict with the rest of the Hai has put pressure on them to gain a technological edge, despite their limited resources. The Ionic Blaster is a result of that pressure. It is much smaller than the venerable Ion Cannon, but is still able to apply the same weapon jamming and energy draining effects, while also doing more raw damage for its size."
```